js/apps/admin-ui/src/identity-providers/add/SamlGeneralSettings.tsx

```diff
@@ -56,7 +56,7 @@ export const SamlGeneralSettings = ({
         >
           <FormattedLink
             title={t("samlEndpointsLabel")}
-            href={`${environment.authServerUrl}/realms/${realm}/broker/${alias}/endpoint/descriptor`}
+            href={`${environment.adminBaseUrl}/realms/${realm}/broker/${alias}/endpoint/descriptor`}
```

Comment:
This was also using the Admin URL before, but I am not entirely sure that was the correct default. This looks like it could be a URL that should point to the regular Keycloak server root.